src/Neo/SmartContract/ApplicationEngine.OpCodePrices.cs

```diff
@@ -75,7 +75,9 @@ partial class ApplicationEngine
             [OpCode.CALLA] = 1 << 9,
             [OpCode.CALLT] = 1 << 15,
             [OpCode.ABORT] = 0,
+            [OpCode.ABORTMSG] = 0,
```

Comment:
It requires a little work with estack to pop message from it, are you sure about the zero price for it?

Reply:
It doesn't really matter, the VM will halt immediately.

Reply:
I think that it should be the same as pop